locales/en/LC_MESSAGES/volto.po

```diff
@@ -2310,7 +2310,7 @@ msgstr "Travel and / or service charges"
 #: components/ItaliaTheme/View/IncaricoView/IncaricoView
 # defaultMessage: Importi di viaggio e/o servizio
 msgid "importi_viaggio"
-msgstr ""
+msgstr "Travel and/or service amounts"
```

Comment:
"Amounts" di cosa? Direi più "expenses" :) se non è specificato nulla dopo "amount" in Inglese, ed è una frase un po' "staccata" dal contesto, non si capisce bene a cosa si riferisce. "Importo" in italiano è quasi sempre riferito ai soldi, ma in inglese "amount" è "quantità", può essere riferito a qualsiasi cosa. Letto così può sembrare "quantità di viaggi e/o servizi". So che poi viene accoppiato ad un prezzo nella View, ma non è comunque un'espressione precisa :)